dotnet/samples/Concepts/Filtering/AutoFunctionInvocationFiltering.cs

```diff
@@ -8,15 +8,18 @@ namespace Filtering;
 
 public class AutoFunctionInvocationFiltering(ITestOutputHelper output) : BaseTest(output)
```

Comment:
Considering that you have a streaming and non-streaming example, I kind of expected them to do the same things, but just differ with regards to streaming and non-streaming, i.e. show how they differ.  I think it might be useful to either show examples that do only differ in that one aspect, or explain what else you are demonstrating using some overall description comment, e.g. "This filter shows how to log function call information in a streaming scenario" and "This filter shows how to skip function invocation in a non-streaming scenario".

Reply:
Thanks, that makes sense. I removed non-streaming/streaming mentioning, since the purpose of new example is to showcase how to get list of function calls, which work in both scenarios.